moto/codepipeline/models.py

```diff
@@ -0,0 +1,151 @@
+import json
+from datetime import datetime
+
+from boto3 import Session
+from moto.core.utils import iso_8601_datetime_with_milliseconds
+
+from moto.iam.exceptions import IAMNotFoundException
+
+from moto.iam import iam_backends
+
+from moto.codepipeline.exceptions import (
+    InvalidStructureException,
+    PipelineNotFoundException,
+    ResourceNotFoundException,
+)
+from moto.core import BaseBackend, BaseModel
+
+DEFAULT_ACCOUNT_ID = "123456789012"
```

Comment:
Can you pull this from `moto.core`? A recent PR added this.

Reply:
Done :)